packages/core/src/components/forms/textArea.tsx

```diff
@@ -50,6 +61,29 @@ export class TextArea extends React.PureComponent<ITextAreaProps, {}> {
             className,
         );
 
-        return <textarea {...htmlProps} className={rootClasses} ref={inputRef} />;
+        const styleProps =
```

Comment:
actually I just realized we should be merging this style object into the one in `htmlProps` if the user provided that. I'll fix it in a follow up